db/schema.rb

```diff
@@ -13,7 +13,7 @@
 ActiveRecord::Schema.define(version: 20180626235055) do
 
   create_table "images", force: :cascade do |t|
-    t.string "link"
+    t.string "link", null: false
```

Comment:
Didn't we make this change in the last pr? Maybe we should just add this as a separate commit. It's generally best to separate every non-related change into separate commits so that we can go back and see what happened, why, and revert individual commits if necessary